CHANGES.rst

```diff
@@ -12,6 +12,23 @@ Yet to be released.
 
 - [Bugfix] #15 #17 - Restore Python 2.6 compatibility. (Thanks to Marco Nenciarini!)
 
+.. attention::
+    Deprecation warnings, these objects (i.e. functions, properties) are
+    slated for removal in the next major release.
```

Comment:
I still think starting a sentence with "Deprecation warnings, these objects ..." makes no sense :wink:

I think "Deprecation warning: These objects ..." would be better.